sandboxes/rslib-react-component/tsconfig.json

```diff
@@ -4,7 +4,10 @@
     "strict": true,
     "resolveJsonModule": true,
     "jsx": "react",
-    "outDir": "dist"
+    "outDir": "dist",
+    "paths": {
+      "*": ["./@mf-types/*"]
```

Comment:
Is any code importing files like `import abd from '*/abc.js'`?

Reply:
no, this is for remote module types. https://module-federation.io/guide/start/quick-start.html#3-consuming-the-producer

Reply:
magic.